frontend/public/config.yml

```
@@ -433,6 +433,11 @@ collections:
           default: 'Odoberať',
           widget: 'string',
         }
+      - {
+          label: 'Link na odoberanie newsletteru',
```

yamllint (1.35.1)

[error] wrong indentation: expected 8 but found 10

(indentation)